app/views/dashboard/collections/_editors.html.erb

```diff
@@ -1,11 +1,11 @@
 <p><%= t('.explanation') %></p>
 
```

Comment:
Can this file be removed? I think you're calling `editors_form` from `collections/edit`